pkg/bundle/kube_version.go

```diff
@@ -0,0 +1,37 @@
+package bundle
+
+import (
+	_ "embed"
```

Comment:
I'm a big fan of the embed package, but I see no use of it, or its directives. Are tehse necessary?

```sh
$ rg -u embed
pkg/bundle/registry_client.go
6:	_ "embed"

pkg/bundle/manager.go
5:	_ "embed"

pkg/bundle/kube_version.go
4:	_ "embed"

$ rg -u go:embed
<no results>
```

Reply:
done